src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private final int bound = 100;
```

Comment:
It is not a constant.
The constant should look like this:
```suggestion
    private static final int BOUND_NUMBER = 100;
```